src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -110,6 +110,8 @@ void GraphingCalculator::GraphingCalculator_DataContextChanged(FrameworkElement
 
     m_variableUpdatedToken = ViewModel->VariableUpdated +=
         ref new EventHandler<VariableChangedEventArgs>(this, &CalculatorApp::GraphingCalculator::OnVariableChanged);
+
+    GraphingNumberPad->TargetEquationInputArea = EquationInputAreaControl;
```

Comment:
It is hard to tell which equation is being edited, especially when it doesn't have focus and is out of the scroll region. Could we scroll it back into view in this scenario, and maybe give it some visual treatment?

Reply:
I like scrolling it back into view.  I'm going to see if we can always show the caret on the active equation input area too.

Reply:
Steph had a good idea which is just to disable the keyboard when a rich edit doesn't have focus.  I think all of the problems go away when that happens and everything is more intuitive.

Reply:
Initial focus is on the graph, which feels wrong.  I think I need to make one more change before this is good to go.

Reply:
after the refactors that went in, this part is much trickier.  We are now just going to no-op when rich edit controls do not have focus.